pkgs/by-name/ca/caddy/package.nix

```
@@ -24,39 +25,42 @@ buildGoModule {
     owner = "caddyserver";
     repo = "caddy";
     rev = "v${version}";
-    hash = "sha256-th0R3Q1nGT0q5PGOygtD1/CpJmrT5TYagrwQR4t/Fvg=";
+    hash = "sha256-/0QRaa2a2OO18dIaWq8ApTfKyroriEo2+tP1mVcvajc=";
   };
 
-  vendorHash = "sha256-ebnSehuhbCY58ctM8IRVMfNxxbJBp6ht9cbuLdGFNek=";
+  vendorHash = "sha256-sdtkgJXakKRL2y+I3M9UJeKsjdIqUeNziEzmlSCUKkA=";
 
   subPackages = [ "cmd/caddy" ];
 
   ldflags = [
-    "-s" "-w"
+    "-s"
+    "-w"
     "-X github.com/caddyserver/caddy/v2.CustomVersion=${version}"
   ];
 
```

Please add the `nobadger` build tag, as mentioned in the release notes and https://github.com/caddyserver/caddy/pull/6031.

Also see https://github.com/caddyserver/xcaddy/commit/d6f5a3e6c3dd820060bd2ec75c8fa67a6cfddb45 and https://github.com/caddyserver/dist/commit/03b7609b4bd9514c0bd84e9b903e9b8573651d78.

```suggestion

  tags = [
    "nobadger"
  ];

```

---

Done, thank you! I undid the `nixfmt-rfc-style` conversion and added the `nobadger` tag.